src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+
+.product-card__stars {
```

Comment:
it's a separate block (even located in a separate file), so you should have used just `stars` selector